python/pyspark/sql/udtf.py

```diff
@@ -82,13 +87,21 @@ class OrderingColumn:
     """
     Represents a single ordering column name for purposes of returning metadata from the 'analyze'
     method.
+
+    Parameters
+    ----------
+    name : str
+        The name of the partitioning column.
+    ascending : bool, default True
+        If this column is in an ascending order or not.
+    overrideNullsFirst : str, optional
+        If this is None, use the default behavior to sort NULL values first when sorting in ascending
+        order, or last when sorting in descending order. Otherwise, if this is True or False, override
+        the default behavior accordingly.
```

Comment:
```suggestion
        If this is None, use the default behavior to sort NULL values first when sorting in
        ascending order, or last when sorting in descending order. Otherwise, if this is
        True or False, override the default behavior accordingly.
```